index.js

```diff
@@ -27,6 +27,9 @@ function fail (err) {
 
 function openConfig (cb) {
   var userHome = require('user-home');
+  if (process.platform !== 'win32' && userHome) {
+    userHome = path.join(userHome, '.cache');
+  }
   var configpath = path.join(userHome || os.tmpdir(), configfile);
```

Comment:
I have a module for exactly this too: https://github.com/sindresorhus/home-or-tmp